sync/block_listener_test.go

```diff
@@ -165,12 +166,12 @@ func TestBlockListener_ListenToGossipBlocks(t *testing.T) {
 	data, err := mesh.BlockAsBytes(*blk)
 	blk2, ok := mesh.BytesAsBlock(data)
 	assert.NoError(t, ok)
-	assert.Equal(t, *blk, blk2)
+	assert.True(t, blk.Compare(&blk2))
 
 	assert.NoError(t, err)
 	n2.Broadcast(NewBlockProtocol, data)
 
-	timeout := time.After(5 * time.Second)
+	timeout := time.After(60 * time.Second)
```

Comment:
I'd reconsider, if somethings not done in 5 seconds in a controlled enviroment this means we have a bug somewhere... on the other hand if somethings not working we shouldn't wait or delay the entire unit tests

Reply:
leftover debugging code ... reverted